man/systemd.exec.xml

```diff
@@ -1341,7 +1347,12 @@ CapabilityBoundingSet=~CAP_B CAP_C</programlisting>
         <programlisting>RuntimeDirectory=foo/bar
 StateDirectory=aaa/bbb ccc</programlisting>
         then the environment variable <literal>RUNTIME_DIRECTORY</literal> is set with <literal>/run/foo/bar</literal>, and
-        <literal>STATE_DIRECTORY</literal> is set with <literal>/var/lib/aaa/bbb:/var/lib/ccc</literal>.</para></listitem>
+        <literal>STATE_DIRECTORY</literal> is set with <literal>/var/lib/aaa/bbb:/var/lib/ccc</literal>.</para>
+
+        <para>Example: if a system service unit has the following,
+        <programlisting>RuntimeDirectory=foo:bar</programlisting>
+        the service manager creates <filename index='false'>/run/foo</filename> (if it does not exist) and
+        <filename index='false'>/run/bar</filename> as a symlink to <filename index='false'>/run/foo</filename>.</para>
```

Comment:
maybe provide example with multiple symlinks at once instead? it's a moderately more complex case but underlines you can have multiple symlinks